dataplane/saiserver/routing_test.go

```diff
@@ -570,6 +570,7 @@ func TestCreateNextHop(t *testing.T) {
 		t.Run(tt.desc, func(t *testing.T) {
 			dplane := &fakeSwitchDataplane{}
 			c, mgr, stopFn := newTestNextHop(t, dplane)
+			mgr.StoreAttributes(10, &saipb.TunnelAttribute{Type: saipb.TunnelType_TUNNEL_TYPE_IPINIP.Enum()})
```

Comment:
From the test cases, 10 is RIF ID and also tunnel ID. So we only store it for tunnel?

Reply:
changed to unique id